misc/gce-federation/README.md

```diff
@@ -0,0 +1,51 @@
+=== GCE quickstart
+cat << EOF > inventory/gce.ini
+gce_service_account_email_address = <service account email address>
+gce_service_account_pem_file_path = <path to service account pem file>
+gce_project_id = <gce project id>
+EOF
+
+export FEDERATION_ID=<my federation id>
```

Comment:
@scollier yes, basically a unique identifier to avoid naming/resource conflicts. I'll add a better explanation with a followup.